docs/sync-storage-handbook/src/logins.md

```diff
@@ -0,0 +1,57 @@
+# Logins
```

Comment:
I noticed that the rust book has a convention of naming their source files like `chXY-title.md`, e.g. `ch00-introduction.md`, so that when you view the files as a directory listing they appear in the same order as they appear in the book. Any thoughts on following a similar convention? (Which if we want to do it, is probably worth doing from the start).